flusher/flusher/handler.py

```diff
@@ -144,6 +150,7 @@ def handle_new_request(self, msg):
         del msg["tx_hash"]
         if "timestamp" in msg:
             self.handle_set_request_count_per_day({"date": msg["timestamp"]})
+            self.handle_set_request_count_per_oracle_script_and_day({"date": msg["timestamp"], "oracle_script_id": msg["oracle_script_id"], "last_update": msg["timestamp"]})
```

Comment:
Are the date and last update use the same msg["timestamp"]? They will auto convert right?

Reply:
yes, it will use type from schema for conversion.